SwiftLeeds/Views/About/AboutView.swift

```diff
@@ -29,6 +33,11 @@ struct AboutView: View {
                 imageAssetName: Assets.Image.swiftLeedsIcon,
                 backgroundImageAssetName: Assets.Image.playhouseImage
             )
+            
+            FancyHeaderView(title: "Steve Jobs",
```

Comment:
Not sure where we're at on linting, but it would be great to tidy this up a little:

```suggestion
FancyHeaderView(
  title: "Steve Jobs",
  foregroundImageURL: foregroundImageURL,
  backgroundImageURL: backgroundImageURL
)
```

The main changes are splitting the args onto new-lines as its a long 'un, and then tidying up the spacing around the commas

Reply:
Done